### PR TITLE
Feature/rooms

### PR DESCRIPTION
**Pull request by FranciscoCano**

### CODE REVIEW

File: OWDTestToolkit/DOM/Loop.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 frame_locator = ('src', 'loop')
 loading_overlay = ('id', 'loading-overlay')
-app_header = ('xpath', '//section[@id="calllog-panel"]//h1[text()="Firefox Hello"]')
+app_header = ('css selector', 'section#calllog-panel')
```
Comment (villenavg) [inline OWDTestToolkit/DOM/Loop.py]: css selector is in da house

File: OWDTestToolkit/apps/email.py [hunk 2]
```diff
@@ -26,6 +27,43 @@ def launch(self):
         self.UTILS.element.waitForNotElements(DOM.GLOBAL.loading_overlay, self.__class__.__name__ + " app - loading overlay")
         return self.app
 
+    def refresh(self):
```
Comment (villenavg) [inline OWDTestToolkit/apps/email.py]: http://i1.ytimg.com/vi/Dgosi8Mp41Q/hqdefault.jpg

Comment (FranciscoCano) [inline OWDTestToolkit/apps/email.py]: :dancers:

File: OWDTestToolkit/apps/loop.py [hunk 3]
```diff
@@ -271,22 +303,13 @@ def phone_login_manually(self, phone_number_without_prefix):
...
-                self.UTILS.element.simulateClick(allow_permission_button)
-
-            _allow_contact_permission()
-
-            self.UTILS.iframe.switchToFrame(*DOM.Loop.mobile_id_frame_locator)
-            time.sleep(2)
-
-            self.UTILS.reporting.logResult('info', "Permissions accepted. Click on verified button!!")
-            self.parent.wait_for_element_displayed(*DOM.Loop.mobile_id_verified_button, timeout=5)
+            self.parent.wait_for_element_displayed(
```
Comment (villenavg) [inline OWDTestToolkit/apps/loop.py]: I think this piece of code does not work right. The code it was before did, though.

File: OWDTestToolkit/apps/loop.py [hunk 4]
```diff
@@ -579,6 +602,141 @@ def toggle_new_communication(self, is_room=True):
...
+        try:
+            self.marionette.switch_to_frame()
+            self.UTILS.reporting.debug("Waiting for Invalid contact message...")
+            self.parent.wait_for_element_displayed(*DOM.Loop.room_share_invalid_contact_ok, timeout=6)
+            self.marionette.find_element(*DOM.Loop.room_share_invalid_contact_ok).tap()
+            return False
+        except:
+            self.UTILS.iframe.switchToFrame(*DOM.Loop.frame_locator)
+            self.UTILS.reporting.debug("Message not found, continue sharing.")
+            pass
```
Comment (villenavg) [inline OWDTestToolkit/apps/loop.py]: not needed anymore

Comment (FranciscoCano) [inline OWDTestToolkit/apps/loop.py]: Oops! right

File: OWDTestToolkit/apps/loop.py (hunk 3 above)
Comment (FranciscoCano) [inline OWDTestToolkit/apps/loop.py]: I think it works, but I will restore it, maybe I've blown something up in the merge...

**Comment by villenavg** [comment]
LGTM

**Pull request merged**